docs/CUSTOMIZING.md

```diff
@@ -0,0 +1,84 @@
+# Customization guide
+
+This document describes how you can create your own app for an event.
+
+## 1. Required information
+
+The following information are needed to configure the app for your event.
+
+- Schedule URL which provides Frab compatible XML
+- Session URL template, e.g. `https://awesome-event.com/2021/events/%1$s.html`
+- Server backend type, one of: `pentabarf`, `frab`, `pretalx`
+- Schedule feedback URL template (optional), e.g. `https://awesome-event.com/2021/events/%s/feedback/new`
+- Google Play URL, e.g. `https://play.google.com/store/apps/details?id=com.awesome.event.schedule`
+- Event URL, e.g. `https://awesome-event.com/2021`
+- Start and end date of the event
+- Email address for bug reports
+- Engelsystem URL (optional), e.g. `https://engelsystem.de/awesome-event/shifts-json-export?key=YOUR_KEY`
+
+## 2. Required graphic assets and colors
+
+The following graphics and colors are needed in order to customize the look and feel of the app.
+
+- Launcher icon (SVG)
+
+  ![Launcher icon examples](gfx/launcher-icon.png)
+
+- Notification icon (SVG)
+
+  ![Notification icon example](gfx/notification-icon.png)
+
+- About dialog logo (SVG)
+
+  ![About dialog logo example](gfx/about-dialog-logo.png)
+
+- Text and background colors (HEX)
```

Comment:
link where you can change that colors in the code?
(haven't looked into code - maybe it's obvious where this is? but it would make it easy for unexperienced devs)

Reply:
(maybe also a link, where the images are in the code used - same as above)

Reply:
The first steps are meant for preparation only. :heavy_check_mark: I clarified this in 4d069023cf26b8e4089bad08b5d1fddb2949727c. Step 3 points out the folder and file names.